monitoring/prometheus/generate-prom.js

```diff
@@ -43,53 +42,74 @@ scrape_configs:
     static_configs:
       - targets: ['localhost:9090']
 
- # monitor docker-compose local setups
+  # monitor data growth
 
-  - job_name: 'local-discovery-provider'
-    metrics_path: '/prometheus_metrics'
+  - job_name: 'load-test-census-stage'
```

Comment:
per our slack discussion, i still don't see why we want this enabled by default on local

Reply:
It's a no-op if it's enabled in any environment and the exporter can't be found. Look at the status of our current [Prometheus targets](http://grafana.audius.co:9090/targets) to gain a better sense of that.

But as we're developing code, I'm okay having exporters that aren't going to run locally unless we enable them, because it doesn't harm anything.

What I'm not okay with is having all remote-dev instances pinging Production exporters when there isn't a need, since that can potentially be harmful.

Additionally, census should be moved from load-test-tools to monitoring/, but I'll go more into detail with a README update and ping you when we enter that topic.

Reply:
> What I'm not okay with is having all remote-dev instances pinging Production exporters when there isn't a need, since that can potentially be harmful.

agreed on this being the biggest concern